pandas/plotting/_core.py

```diff
@@ -2753,20 +2767,30 @@ def hist(self, by=None, bins=10, **kwds):
         """
         return self(kind='hist', by=by, bins=bins, **kwds)
 
-    def kde(self, **kwds):
+    def kde(self, bw_method=None, ind=None, **kwds):
         """
         Kernel Density Estimate plot
 
         Parameters
         ----------
+        bw_method: str, scalar or callable, optional
+            The method used to calculate the estimator bandwidth.  This can be
+            'scott', 'silverman', a scalar constant or a callable.
+            If None (default), 'scott' is used.
+            See :class:`scipy.stats.gaussian_kde` for more information.
+        ind : NumPy array or integer, optional
+            Evaluation points. If None (default), 1000 equally spaced points
+            are used. If `ind` is a NumPy array, the kde is evaluated at the
+            points passed. If `ind` is an integer, `ind` number of equally
+            spaced points are used.
         `**kwds` : optional
-            Keyword arguments to pass on to :py:meth:`pandas.DataFrame.plot`.
+            Keyword arguments to pass on to :py:meth:`pandas.Series.plot`.
```

Comment:
is this change right? This the DataFrame one

Reply:
Good catch. My bad. It should be ``pandas.DataFrame.plot``, and I have made the change.